utility/doc/E2E_FEATURE_LIST.md

```diff
@@ -0,0 +1,121 @@
+# E2E Features <!-- omit in toc -->
+
+- [Legend](#legend)
+- [A. E2E Relay](#a-e2e-relay)
+  - [1. Trustless Relay 🟡](#1-trustless-relay-)
+  - [2. Delegated Relay 🟡](#2-delegated-relay-)
+  - [3. Rate Limited Application 🟢🔴](#3-rate-limited-application-)
+- [B. E2E QoS](#b-e2e-qos)
```

Comment:
cut all quality of service checks on protocol from v1 scope (imo)

Reply:
Which of these would you recommend:

1. Make them nice-to-haves
2. Remove them completely from this dock
3. Add a post-launch emoji and mark it as that

Reply:
i like option 3, let's push it to v1.X. i think it's important to continue to shape the roadmap and let everyone know that's the goal, just that we need to get to V1 faster and then iterate